src/devices/Gpio/README.md

```diff
@@ -1,80 +1,34 @@
-# Sunxi GPIO Driver for .NET
+# GPIO Driver for Special Chip
```

Comment:
I think this label will look really weird on the device listing. I'd recommend to change to something like:

```suggestion
# GpioDriver for other boards
```